automation_tools/scripts/ga-migration/README.md

```diff
@@ -1,47 +1,74 @@
 # ga-migration
 
+This folder contains the scripts used to migrate a repository from Travis CI to GitHub actions.
+
+The migration pipeline is *idempotent*, meaning you should be able to run it how many times you want, getting the same results.
+
+The `utils.py` file contains the tasks implementations, while `main.py` imports and applies them according to our needs. `gitflow.py` handles the higher level git and GitHub procedures.
+
 ## Install
 
-```console
-$ virtualenv ~/.virtualenvs/ga-migration
-$ source ~/.virtualenvs/ga-migration/bin/activate
-$ pip install -r requirements.txt
+```bash
+# Activate a virtualenv and install pip requirements
+virtualenv ~/.virtualenvs/ga-migration
+source ~/.virtualenvs/ga-migration/bin/activate
+pip install -r requirements.txt
 ```
 
 ## Migration
 
-Apply migration patches on a cloned repository:
+To apply migration patches on a cloned repository, run `python main.py --targetpath=invenio-charts-js`.
+
+E.g.:
 
 ```bash
-# e.g.
-# git clone https://github.com/inveniosoftware/invenio-charts-js
+# cd to the scripts directory
+cd automation-tools/scripts/ga-migration
+# clone the repository to migrate
+git clone https://github.com/inveniosoftware/invenio-charts-js
```

Comment:
```suggestion
# clone the repository to migrate
git clone https://github.com/inveniosoftware/invenio-charts-js

# cd to the scripts directory
cd automation-tools/scripts/ga-migration
```

I suggest reversing the order and you can make two different directories in starting.

```
mkdir repo_folder
mkdir script_folder

git clone repo and git clone script in their respective directories

```

By doing this, you will be able to show the `--targetpath` in the example easily.

Reply:
This should be clearer now